### ISSUE TITLE
WoE 2 Barricades

### ISSUE THREAD

**Issue by gmsour** [rathena/rathena#1760]
Over Brand will damage barricade even when the Guardian Stones are intact

**Comment by anacondaqq** [comment]
Hi, if you wish to fix the issue, please spend a few more minutes and provider better information about the issue like:

1. Your git hash
2. How reproduce
3. Do you have any custom source edits at your server?
4. If yes, have you tried to reproduce the bug on clean rAthena?
5. Optional: client date / packet version / server mechanic

This is important for developers to have exact the same things like you have, to reproduce the bug (if it exist)

**Comment by Darkelfen** [comment]
I've got the same issue with the barricades on schg_cas01 which are damageable even though the Guardian Stones are intact.

Mode: renewal.
Githash: Aproximately https://github.com/rathena/rathena/commit/561a874833ae0aac0301667447d144071ca76afc
Client: 2015-11-04

How to reproduce: 
Activate WoE on this castle and make sure both guardian stones are up.
Go to the first barricade, (the one straight from the entrance) try to damage the barricades with skills.

I've noticed that not only Overbrand but other skills as well can damage the barricades when they should be in an "inmune state" until both Guardian Stones are destroyed.
